tendrl-node-agent.spec

```diff
@@ -90,8 +91,12 @@ py.test -v tendrl/node-agent/tests || :
 %{_unitdir}/tendrl-node-agent.service
 %{_unitdir}/tendrl-node-agent.socket
 %config(noreplace) %{_sysconfdir}/rsyslog.d/tendrl-node-agent.conf
+%{_sysconfdir}/logrotate.d/tendrl-node-agent_logrotate.conf
```

Comment:
i am not giving %config(noreplace) because while removing node-agent it should be removed I feel, otherwise next time while installing node-agent we will see two log rotation files.

Reply:
Ack. I feel tendrl should have control of when to rotate its logs files.

Reply:
@GowthamShanmugam I feel logrotate daemon doesnt consider files with .rpmsave so we can have %config(noreplace) %{_sysconfdir}/logrotate.d/tendrl-node-agent_logrotate.conf

https://forums.fedoraforum.org/archive/index.php/t-298877.html